resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/ratelimiter/monitoring/health/RateLimitersHealthIndicator.java

```diff
@@ -55,6 +57,12 @@ public Health health() {
     private boolean isRegisterHealthIndicator(RateLimiter rateLimiter) {
         return rateLimiterProperties.findRateLimiterProperties(rateLimiter.getName())
                 .map(InstanceProperties::getRegisterHealthIndicator)
+                .orElse(true);
```

Comment:
We decided to disable HealthIndicator by default. Please set `false` the default value.